### PR TITLE
Installing on Wildfly

### PR DESCRIPTION
**Pull request by codyhoag**
Sending this to you for SME review. Let me know if you have any questions! Thanks!

https://issues.liferay.com/browse/LRDOCS-4731

### CODE REVIEW

**Comment by dantewang** [comment]
Fetched <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: discover/deployment/articles/00-deploying-liferay/05-installing-liferay-portal-on-wildfly.markdown [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+    this:
+
+        <jsp-config development="true" source-vm="1.8" target-vm="1.8" />
+
+**Checkpoint:**
+
+Before continuing, verify the following properties have been set in the
+`standalone.xml` file:
+
+1.  The new `<system-property>` is created.
```
Comment (dantewang) [inline discover/deployment/articles/00-deploying-liferay/05-installing-liferay-portal-on-wildfly.markdown:251]: @codyhoag In line 198 we are adding `<system-properties>` element with `<property>` child elements; I think it's better to say "The new `<system-properties>` element is added` to be consistent.

I may miss this part in the JBoss doc.

cc @alee8888 Other things look good to me.

Comment (codyhoag) [reply discover/deployment/articles/00-deploying-liferay/05-installing-liferay-portal-on-wildfly.markdown:251]: @dantewang makes sense; I'll update it. Thanks!

**Comment by codyhoag** [comment]
@alee8888 since Dante reviewed this doc, is it OK to send this on for publication?

**Comment by alee8888** [comment]
@codyhoag yes please push this through. Thanks!

**Pull request closed without merging**